tests/test_date.py

```diff
@@ -2,90 +2,158 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from types import MethodType
+
+from collections import OrderedDict
+
 import unittest
 from datetime import datetime, timedelta
 
 from dateutil.relativedelta import relativedelta
 from mock import Mock, patch
+
 from nose_parameterized import parameterized, param
 
+from dateparser.languages import LanguageDataLoader
+
 from dateparser import date
 from tests import BaseTestCase
 
 
-class DateRangeTest(unittest.TestCase):
+class DateRangeTest(BaseTestCase):
+    def setUp(self):
+        super(DateRangeTest, self).setUp()
```

Comment:
You can omit this method if the only thing you need from it is to call `super()`. Although in this case please set the attributes you assign during test to `NotImplemented` to be sure that any part of test state would not be applied to the next one test.